cockatrice/src/chatview.cpp

```diff
@@ -155,12 +156,14 @@ void ChatView::appendMessage(QString message, RoomMessageTypeFlags messageType,
         senderFormat.setAnchorHref("user://" + QString::number(userLevel) + "_" + sender);
         if (sameSender) {
             cursor.insertText("    ");
-        } else {
+        }
+        else {
             if (!sender.isEmpty() && tabSupervisor->getUserListsTab()) {
                 const int pixelSize = QFontInfo(cursor.charFormat().font()).pixelSize();
                 QMap<QString, UserListTWI *> buddyList = tabSupervisor->getUserListsTab()->getBuddyList()->getUsers();
-                cursor.insertImage(UserLevelPixmapGenerator::generatePixmap(pixelSize, userLevel,
-                                                                            buddyList.contains(sender)).toImage());
+                QMap<QString, UserListTWI *> userList = tabSupervisor->getUserListsTab()->getAllUsersList()->getUsers();
```

Comment:
This works fine, but it will need some refactoring in the future; right now we are retrieving data from the `QGroupBox` in the userlist tab, instead we should have a generic "connection" object that provides this data.
This is related to point 2 discussed in PR https://github.com/Cockatrice/Cockatrice/pull/2267